pysat/_instrument.py

```diff
@@ -523,6 +519,10 @@ def __init__(self, platform=None, name=None, tag='', inst_id='',
         # Store base attributes, used in particular by Meta class
         self._base_attr = dir(self)
 
+        # Run instrument init function, a basic pass function is used if the
```

Comment:
Handeld more completelty in #987

Reply:
Also need the functionality in #987 to pass here. Going to go ahead and merge that in so testing can run.